pkg/remote/deploy_pfe.go

```diff
@@ -178,21 +195,57 @@ func setPFEEnvVars(codewind Codewind, deployOptions *DeployOptions) []corev1.Env
 	}
 }
 
-// setPFEVolumes returns the 3 volumes & corresponding volume mounts required by the PFE container:
+func createCodewindPVC(codewind Codewind, deployOptions *DeployOptions, storageClass string) corev1.PersistentVolumeClaim {
+
+	labels := map[string]string{
+		"app":               KeycloakPrefix,
+		"codewindWorkspace": codewind.WorkspaceID,
+	}
+
+	pvc := corev1.PersistentVolumeClaim{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "PersistentVolumeClaim",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   PFEPrefix + "-pvc-" + codewind.WorkspaceID,
+			Labels: labels,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				"ReadWriteMany",
+			},
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse("1Gi"),
```

Comment:
Also have some concerns about setting the size of the volume to a hardcoded value (which we currently can't avoid with the code in Che).

Could we make this configurable with some kind of flag (`--pvc-size`?)

Reply:
Added an option to set a the size and updated output message during install to show additional info : 

```
INFO[0095] Creating Codewind PVC codewind-pfe-pvc-k3j227kc to 2Gi  
```